.eslintrc.js

```diff
@@ -65,6 +65,20 @@ module.exports = {
       functions: 'never',
     }],
 
+    'jsdoc/require-file-overview': [2, {
+      tags: {
+        file: {
```

Comment:
just to make it clear? the rule name is rather misleading

```suggestion
        file: {
          mustExist: false,
```